src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java

```diff
@@ -353,4 +356,23 @@ public static void loadImageFromFont(Context context, @ColorInt int color, Strin
             .override(Target.SIZE_ORIGINAL, Target.SIZE_ORIGINAL)
             .into(target);
     }
+
+    public static final ColorStateList getColorStateListForStyleableAttribute(Context context, String styleable, int resId, int index) {
+        try {
+            Field[] fields = R.styleable.class.getFields();
+            Field field = R.styleable.class.getField(styleable);
+            int [] attributes = (int[]) field.get(R.styleable.class);
+            TintTypedArray styledAttributes =  TintTypedArray.obtainStyledAttributes (context, null, attributes, resId, 0);
+            try {
+                return styledAttributes.getColorStateList (index);
+            } finally {
+                styledAttributes.recycle ();
+            }
+        } catch (NoSuchFieldException e) {
+            e.printStackTrace();
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
+        }
```

Comment:
I would remove these, and maybe just let them throw?

I think you fixed the problem here by adding the resource.